test/cases/Variadic.idl

```diff
@@ -0,0 +1,10 @@
+interface Variadic {
+  void simple1(DOMString... strings);
+  void simple2(DOMString first, URL... urls);
```

Comment:
Can you add a space between these so that only overloads are grouped together in this file?